paasta_tools/check_marathon_services_replication.py

```diff
@@ -198,12 +193,13 @@ def send_event_if_under_replication(
     service,
     instance,
     cluster,
-    crit_threshold,
     expected_count,
     num_available,
     soa_dir,
 ):
     full_name = compose_job_id(service, instance)
+    job_config = marathon_tools.load_marathon_service_config(service, instance, cluster)
```

Comment:
Seems like we should be looking this up in check_healthy_marathon_tasks_for_service_instance; ideally we'd avoid re-reading the marathon.yaml over and over.